pages/_app.tsx

```diff
@@ -12,10 +13,13 @@ function MyApp({ Component, pageProps }: AppProps) {
         <title>Roseate Cards</title>
         <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap" rel="stylesheet" />
```

Comment:
Can we add a todo/Github issue to add meta descriptions and things for social media link sharing?

Reply:
Do you want to add it to your notes?